sdk/core/core-paging/src/models.ts

```diff
@@ -31,3 +31,22 @@ export interface PagedAsyncIterableIterator<T, PageT = T[], PageSettingsT = Page
    */
   byPage: (settings?: PageSettingsT) => AsyncIterableIterator<PageT>;
 }
+
+/**
+ * An interface that describes how to communicate with the service.
+ */
+export interface PagedResult<TPage, TPageSettings = PageSettings> {
+  /**
+   * Link to the first page of results.
+   */
+  link: string;
```

Comment:
I know we call it `nextLink` in payloads, but I wonder if something with `url` in it is more appropriate here, since that's what the string really is, right?

```suggestion
  firstPageUrl: string;
```

Reply:
Done.